WordPress/twentytwentyfive
# Replace the image of the woman in the canoe

**Description**
Replace the image of the woman in the canoe with the portrait of the African woman in the traditional dress.

**Testing Instructions**
Add the "Instagram grid" pattern. Confirm that all images work and that there are no block validation errors.
